common/main/java/com/couchbase/lite/internal/exec/ConcurrentExecutor.java

```diff
@@ -54,6 +54,13 @@ class ConcurrentExecutor implements ExecutionService.CloseableExecutor {
         this.executor = executor;
     }
 
+    /**
+     * Get the number of tasks awaiting execution.
+     *
+     * @return the number of tasks awaiting execution.
+     */
+    public int getPending() { return executor.getQueue().size(); }
```

Comment:
Is the queue.size() thread safe?

Reply:
Yeah... Verified.  Might need a comment though....

Java's BlockingQueue is thread-safe.